### ISSUE TITLE
Currently Estuary release is issuing deals that are too small

### ISSUE THREAD

**Issue by jlogelin** [application-research/estuary#386]
**Describe the bug**
Estuary started sending much smaller deals, compared to 1 month ago — 66.6 MiB, sometimes less than 1 MiB, etc. 66 MiB is wasteful, given that these deals are included in 128 MiB pieces (pretty much 2x overhead). Likely a missed regression in the recent release.

**To Reproduce**
1. Upload 100-200MiB of data
2. Wait to be picked up by deals

**Comment by nonsense** [comment]
@jlogelin this is still happening. I continue to receive deals that are 66.6 MiB or even less than 1 MiB in size, over the last 12h.

**Comment by jlogelin** [comment]
@nonsense hmm - that's odd, we rolled back to v0.1.7 yesterday, so I would have expected this issue to be resolved. Question: approximately how long have you been experiencing these smaller deals?

**Comment by en0ma** [comment]
@nonsense can you provide deals UUIDs we can look at?

**Comment by nonsense** [comment]
From 28m ago.

```
=== Deal cff2dd1b-9b80-4902-b958-135b945bfc39 ===

CreatedAt: 2022-08-04 16:49:52.741 (28m ago)
Client Address: f3vnq2cmwig3qjisnx5hobxvsd4drn4f54xfxnv4tciw6vnjdsf5xipgafreprh5riwmgtcirpcdmi3urbg36a
Client Peer ID: 12D3KooWCVXs8P7iq6ao4XhfAmKWrEeuKFWCJgqe9jGDMTqHYBjw
Signed Proposal CID: bafyreieh3cdqeoopltnfgy7ey6qf7hkdi2c43rm7yaft6shjppbkjhxeoi
Deal Data Root CID: bafybeiauucbauvarjagwpkb7guth35dose5x2x4uwvbmwn6j3ps3abfeie
Verified: Yes
Piece CID: baga6ea4seaqj7iwncqupvlbqx4ahtsjxd7bzddywzimgpjgndkuc7z6izepeipq
Piece Size: 128 MiB (134,217,728 bytes)
Client Collateral: 0 atto
Provider Collateral: 27,969 nano
Storage Price / epoch / GiB: 0 atto
Current Epoch: 2,043,875
Start Epoch: 2,063,979 (in 7 days)
End Epoch: 3,558,699 (in a year)
Duration: 1,494,720 (518 days)
Storage Fee: 0 atto (Price per epoch x Piece Size x Duration)
Transfer Mode: Online
Transfer Type: libp2p
Transfer Size: 66.6 MiB (69,876,450 bytes)
Transfer Params: URL: libp2p:///ip4/3.134.223.177/tcp/6745/p2p/12D3KooWN8vAoGd6eurUSidcpLYguQiGZwt4eVgDvbgaS7kiGTup
Transferred: 66.6 MiB (69,876,450 bytes)
Inbound File Path: /root/.boost/incoming/cff2dd1b-9b80-4902-b958-135b945bfc39.download
Publish Message CID: bafy2bzacedzldy3nqg5oa2iuljrrt4vz3kfxumgxdbrnfk3g2xf4r6vsknzqq
Chain Deal ID: 8,578,220
Checkpoint: PublishConfirmed (3s ago)
Status: Adding to Sector

=== Logs ===
2022-08-04 16:49:52.538: executing deal proposal received from network
{
  "peer": "12D3KooWCVXs8P7iq6ao4XhfAmKWrEeuKFWCJgqe9jGDMTqHYBjw"
}
2022-08-04 16:49:52.554: processing deal acceptance request
2022-08-04 16:49:52.709: tagged funds for deal
{
  "tagged for deal publish": "1050000000000000000",
  "tagged for deal collateral": "27969000276719",
  "total tagged for publish": "6800000000000000000",
  "total tagged for collateral": "12746499997006788",
  "total available for publish": "35637422381300898332",
  "total available for collateral": "21961343616360800453"
}
2022-08-04 16:49:52.733: created deal download staging file
{
  "path": "/root/.boost/incoming/cff2dd1b-9b80-4902-b958-135b945bfc39.download"
}
2022-08-04 16:49:52.749: inserted deal into deals DB
2022-08-04 16:49:52.757: deal execution initiated
{
  "deal state": {
    "DealUuid": "cff2dd1b-9b80-4902-b958-135b945bfc39",
    "CreatedAt": "2022-08-04T16:49:52.741199189+03:00",
    "ClientDealProposal": {
      "Proposal": {
        "PieceCID": {
          "/": "baga6ea4seaqj7iwncqupvlbqx4ahtsjxd7bzddywzimgpjgndkuc7z6izepeipq"
        },
        "PieceSize": 134217728,
        "VerifiedDeal": true,
        "Client": "f3vnq2cmwig3qjisnx5hobxvsd4drn4f54xfxnv4tciw6vnjdsf5xipgafreprh5riwmgtcirpcdmi3urbg36a",
        "Provider": "f0127896",
        "Label": "mAXASIBSgggpUEUgNZ6g/NSZ99G6RO31flLVCyzfJ2+WwBKRB",
        "StartEpoch": 2063979,
        "EndEpoch": 3558699,
        "StoragePricePerEpoch": "0",
        "ProviderCollateral": "27969000276719",
        "ClientCollateral": "0"
      },
      "ClientSignature": {
        "Type": 0,
        "Data": null
      }
    },
    "IsOffline": false,
    "ClientPeerID": "12D3KooWCVXs8P7iq6ao4XhfAmKWrEeuKFWCJgqe9jGDMTqHYBjw",
    "DealDataRoot": {
      "/": "bafybeiauucbauvarjagwpkb7guth35dose5x2x4uwvbmwn6j3ps3abfeie"
    },
    "InboundFilePath": "/root/.boost/incoming/cff2dd1b-9b80-4902-b958-135b945bfc39.download",
    "Transfer": {
      "Type": "libp2p",
      "ClientID": "887444",
      "Params": null,
      "Size": 69876450
    },
    "ChainDealID": 0,
    "PublishCID": null,
    "SectorID": 0,
    "Offset": 0,
    "Length": 0,
    "Checkpoint": 0,
    "CheckpointAt": "2022-08-04T16:49:52.741199319+03:00",
    "Err": "",
    "Retry": "auto",
    "NBytesReceived": 0
  }
}
2022-08-04 16:49:52.757: deal accepted and scheduled for execution
2022-08-04 16:49:52.776: size of downloaded file
{
  "filepath": "/root/.boost/incoming/cff2dd1b-9b80-4902-b958-135b945bfc39.download",
  "size": 0
}
2022-08-04 16:49:52.784: deal execution in progress
2022-08-04 16:49:52.796: transferring deal data
{
  "transfer client id": "887444"
}
2022-08-04 16:49:52.803 [http-transport]: execute transfer
{
  "deal size": 69876450,
  "output file": "/root/.boost/incoming/cff2dd1b-9b80-4902-b958-135b945bfc39.download",
  "time before context deadline": "23h59m59.99999411s"
}
2022-08-04 16:49:52.811 [http-transport]: existing file size
{
  "file size": 0,
  "deal size": 69876450
}
2022-08-04 16:49:52.819 [http-transport]: libp2p-http url
{
  "url": "libp2p://12D3KooWN8vAoGd6eurUSidcpLYguQiGZwt4eVgDvbgaS7kiGTup",
  "peer id": "12D3KooWN8vAoGd6eurUSidcpLYguQiGZwt4eVgDvbgaS7kiGTup",
  "multiaddr": "/ip4/3.134.223.177/tcp/6745"
}
2022-08-04 16:49:52.826 [http-transport]: started async http transfer
2022-08-04 16:49:52.827 [http-transport]: sending http request
{
  "received": 0,
  "remaining": 69876450,
  "range-rq": "bytes=0-"
}
2022-08-04 16:49:57.817: transfer progress
{
  "bytes received": 7016082,
  "deal size": 69876450,
  "percent complete": 10
}
2022-08-04 16:50:02.954: transfer progress
{
  "bytes received": 13995939,
  "deal size": 69876450,
  "percent complete": 20
}
2022-08-04 16:50:08.333: transfer progress
{
  "bytes received": 20975757,
  "deal size": 69876450,
  "percent complete": 30
}
2022-08-04 16:50:14.229: transfer progress
{
  "bytes received": 27955614,
  "deal size": 69876450,
  "percent complete": 40
}
2022-08-04 16:50:19.040: transfer progress
{
  "bytes received": 34968239,
  "deal size": 69876450,
  "percent complete": 50
}
2022-08-04 16:50:21.784: transfer progress
{
  "bytes received": 41948057,
  "deal size": 69876450,
  "percent complete": 60
}
2022-08-04 16:50:24.781: transfer progress
{
  "bytes received": 48927914,
  "deal size": 69876450,
  "percent complete": 70
}
2022-08-04 16:50:27.712: transfer progress
{
  "bytes received": 55907771,
  "deal size": 69876450,
  "percent complete": 80
}
2022-08-04 16:50:30.631: transfer progress
{
  "bytes received": 62920357,
  "deal size": 69876450,
  "percent complete": 90
}
2022-08-04 16:50:34.195 [http-transport]: http server sent EOF
{
  "received": 69876450,
  "deal-size": 69876450
}
2022-08-04 16:50:34.195: transfer progress
{
  "bytes received": 69876450,
  "deal size": 69876450,
  "percent complete": 100
}
2022-08-04 16:50:34.210 [http-transport]: http transfer completed successfully
2022-08-04 16:50:34.223 [http-transport]: http request finished successfully
{
  "nBytesReceived": 69876450,
  "file size": 69876450
}
2022-08-04 16:50:34.231: deal data-transfer completed successfully
{
  "bytes received": 69876450,
  "time taken": "41.427597634s"
}
2022-08-04 16:50:34.239: checking commP
2022-08-04 16:50:35.537: commP matched successfully: deal-data verified
2022-08-04 16:50:35.555: updated deal checkpoint in DB
{
  "old checkpoint": "Accepted",
  "new checkpoint": "Transferred"
}
2022-08-04 16:50:35.564: deal data transfer finished successfully
2022-08-04 16:50:35.572: deal data-transfer can no longer be cancelled
2022-08-04 16:50:35.584: sending deal to deal publisher
2022-08-04 17:10:37.033: updated deal checkpoint in DB
{
  "old checkpoint": "Transferred",
  "new checkpoint": "Published"
}
2022-08-04 17:10:37.041: deal published successfully, will await deal publish confirmation
2022-08-04 17:10:37.049: awaiting deal publish confirmation
2022-08-04 17:14:17.649: error while waiting for publish confirm
{
  "err": "WaitForPublishDeals errored: handler: websocket connection closed"
}
2022-08-04 17:14:17.720: deal paused because of recoverable error
{
  "err": "wait for confirmation of publish message bafy2bzacedzldy3nqg5oa2iuljrrt4vz3kfxumgxdbrnfk3g2xf4r6vsknzqq failed: WaitForPublishDeals errored: handler: websocket connection closed",
  "checkpoint": "Published",
  "retry": "auto"
}
2022-08-04 17:14:17.833: deal go-routine finished execution
2022-08-04 17:17:48.929: user initiated deal retry
{
  "checkpoint": 2
}
2022-08-04 17:17:48.929: deal execution initiated
{
  "deal state": {
    "DealUuid": "cff2dd1b-9b80-4902-b958-135b945bfc39",
    "CreatedAt": "2022-08-04T16:49:52.741199189+03:00",
    "ClientDealProposal": {
      "Proposal": {
        "PieceCID": {
          "/": "baga6ea4seaqj7iwncqupvlbqx4ahtsjxd7bzddywzimgpjgndkuc7z6izepeipq"
        },
        "PieceSize": 134217728,
        "VerifiedDeal": true,
        "Client": "f3vnq2cmwig3qjisnx5hobxvsd4drn4f54xfxnv4tciw6vnjdsf5xipgafreprh5riwmgtcirpcdmi3urbg36a",
        "Provider": "f0127896",
        "Label": "mAXASIBSgggpUEUgNZ6g/NSZ99G6RO31flLVCyzfJ2+WwBKRB",
        "StartEpoch": 2063979,
        "EndEpoch": 3558699,
        "StoragePricePerEpoch": "0",
        "ProviderCollateral": "27969000276719",
        "ClientCollateral": "0"
      },
      "ClientSignature": {
        "Type": 0,
        "Data": null
      }
    },
    "IsOffline": false,
    "ClientPeerID": "12D3KooWCVXs8P7iq6ao4XhfAmKWrEeuKFWCJgqe9jGDMTqHYBjw",
    "DealDataRoot": {
      "/": "bafybeiauucbauvarjagwpkb7guth35dose5x2x4uwvbmwn6j3ps3abfeie"
    },
    "InboundFilePath": "/root/.boost/incoming/cff2dd1b-9b80-4902-b958-135b945bfc39.download",
    "Transfer": {
      "Type": "libp2p",
      "ClientID": "",
      "Params": null,
      "Size": 69876450
    },
    "ChainDealID": 0,
    "PublishCID": {
      "/": "bafy2bzacedzldy3nqg5oa2iuljrrt4vz3kfxumgxdbrnfk3g2xf4r6vsknzqq"
    },
    "SectorID": 0,
    "Offset": 0,
    "Length": 0,
    "Checkpoint": 2,
    "CheckpointAt": "2022-08-04T17:10:37.025273734+03:00",
    "Err": "wait for confirmation of publish message bafy2bzacedzldy3nqg5oa2iuljrrt4vz3kfxumgxdbrnfk3g2xf4r6vsknzqq failed: WaitForPublishDeals errored: handler: websocket connection closed",
    "Retry": "auto",
    "NBytesReceived": 0
  }
}
2022-08-04 17:17:48.972: size of downloaded file
{
  "filepath": "/root/.boost/incoming/cff2dd1b-9b80-4902-b958-135b945bfc39.download",
  "size": 69876450
}
2022-08-04 17:17:48.985: deal execution in progress
2022-08-04 17:17:48.999: deal data transfer has already been completed
2022-08-04 17:17:49.013: deal data-transfer can no longer be cancelled
2022-08-04 17:17:49.030: deal has already been published
2022-08-04 17:17:49.043: awaiting deal publish confirmation
2022-08-04 17:17:49.180: deal publish confirmed
2022-08-04 17:17:49.200: updated deal checkpoint in DB
{
  "old checkpoint": "Published",
  "new checkpoint": "PublishConfirmed"
}
2022-08-04 17:17:49.210: deal successfully published and confirmed-publish
2022-08-04 17:17:49.248: funds successfully untagged for deal after publish
2022-08-04 17:17:49.265: add piece called

```

**Comment by nonsense** [comment]
> Question: approximately how long have you been experiencing these smaller deals?

For the past 5-6 days.

**Comment by nonsense** [comment]
This is still happening. For example (from 9h ago):

```
=== Deal d1e56dbf-fe49-4307-bd6a-65c488d93e7b ===

CreatedAt: 2022-08-11 02:07:57.026 (9h ago)
Client Address: f3vnq2cmwig3qjisnx5hobxvsd4drn4f54xfxnv4tciw6vnjdsf5xipgafreprh5riwmgtcirpcdmi3urbg36a
Client Peer ID: 12D3KooWCVXs8P7iq6ao4XhfAmKWrEeuKFWCJgqe9jGDMTqHYBjw
Signed Proposal CID: bafyreicbkrqlwch55koyf6d6tpxy4vtzgxzmd3pgg3dctcq4wajck6f3bm
Deal Data Root CID: QmReqeDwDdEcsgkNVTsv4YTT7P6RoqaXXEVns5uWmq3yka
Verified: Yes
Piece CID: baga6ea4seaqmoiyedfzisjfesu6aebw254u2z6qyniwvkpibp3hkwa7wrcussdq
Piece Size: 512 KiB (524,288 bytes)
Client Collateral: 0 atto
Provider Collateral: 110.4 nano
Storage Price / epoch / GiB: 0 atto
Current Epoch: 2,063,408
Start Epoch: 2,082,495 (in 7 days)
End Epoch: 3,577,215 (in a year)
Duration: 1,494,720 (518 days)
Storage Fee: 0 atto (Price per epoch x Piece Size x Duration)
Transfer Mode: Online
Transfer Type: libp2p
Transfer Size: 365 KiB (374,105 bytes)
Transfer Params: URL: libp2p:///ip4/139.178.68.217/tcp/6745/p2p/12D3KooWCVXs8P7iq6ao4XhfAmKWrEeuKFWCJgqe9jGDMTqHYBjw
Transferred: 0 B (0 bytes)
Inbound File Path: /root/.boost/incoming/d1e56dbf-fe49-4307-bd6a-65c488d93e7b.download
Publish Message CID: 
Chain Deal ID: 
Checkpoint: Complete (9h ago)
Status: Error: user manually terminated the deal

=== Logs ===
2022-08-11 02:07:56.820: executing deal proposal received from network
{
  "peer": "12D3KooWCVXs8P7iq6ao4XhfAmKWrEeuKFWCJgqe9jGDMTqHYBjw"
}
2022-08-11 02:07:56.836: processing deal acceptance request
2022-08-11 02:07:56.993: tagged funds for deal
{
  "tagged for deal publish": "1050000000000000000",
  "tagged for deal collateral": "110400374015",
  "total tagged for publish": "21050000000000000000",
  "total tagged for collateral": "28882556822678609",
  "total available for publish": "7905559765114874020",
  "total available for collateral": "22059416929461455095"
}
2022-08-11 02:07:57.018: created deal download staging file
{
  "path": "/root/.boost/incoming/d1e56dbf-fe49-4307-bd6a-65c488d93e7b.download"
}
2022-08-11 02:07:57.035: inserted deal into deals DB
2022-08-11 02:07:57.042: deal accepted and scheduled for execution
2022-08-11 02:07:57.043: deal execution initiated
{
  "deal state": {
    "DealUuid": "d1e56dbf-fe49-4307-bd6a-65c488d93e7b",
    "CreatedAt": "2022-08-11T03:07:57.026844189+03:00",
    "ClientDealProposal": {
      "Proposal": {
        "PieceCID": {
          "/": "baga6ea4seaqmoiyedfzisjfesu6aebw254u2z6qyniwvkpibp3hkwa7wrcussdq"
        },
        "PieceSize": 524288,
        "VerifiedDeal": true,
        "Client": "f3vnq2cmwig3qjisnx5hobxvsd4drn4f54xfxnv4tciw6vnjdsf5xipgafreprh5riwmgtcirpcdmi3urbg36a",
        "Provider": "f0127896",
        "Label": "QmReqeDwDdEcsgkNVTsv4YTT7P6RoqaXXEVns5uWmq3yka",
        "StartEpoch": 2082495,
        "EndEpoch": 3577215,
        "StoragePricePerEpoch": "0",
        "ProviderCollateral": "110400374015",
        "ClientCollateral": "0"
      },
      "ClientSignature": {
        "Type": 0,
        "Data": null
      }
    },
    "IsOffline": false,
    "ClientPeerID": "12D3KooWCVXs8P7iq6ao4XhfAmKWrEeuKFWCJgqe9jGDMTqHYBjw",
    "DealDataRoot": {
      "/": "QmReqeDwDdEcsgkNVTsv4YTT7P6RoqaXXEVns5uWmq3yka"
    },
    "InboundFilePath": "/root/.boost/incoming/d1e56dbf-fe49-4307-bd6a-65c488d93e7b.download",
    "Transfer": {
      "Type": "libp2p",
      "ClientID": "973468",
      "Params": null,
      "Size": 374105
    },
    "ChainDealID": 0,
    "PublishCID": null,
    "SectorID": 0,
    "Offset": 0,
    "Length": 0,
    "Checkpoint": 0,
    "CheckpointAt": "2022-08-11T03:07:57.026844309+03:00",
    "Err": "",
    "Retry": "auto",
    "NBytesReceived": 0
  }
}
2022-08-11 02:07:57.066: size of downloaded file
{
  "filepath": "/root/.boost/incoming/d1e56dbf-fe49-4307-bd6a-65c488d93e7b.download",
  "size": 0
}
2022-08-11 02:07:57.075: deal execution in progress
2022-08-11 02:07:57.085: transferring deal data
{
  "transfer client id": "973468"
}
2022-08-11 02:07:57.093 [http-transport]: execute transfer
{
  "deal size": 374105,
  "output file": "/root/.boost/incoming/d1e56dbf-fe49-4307-bd6a-65c488d93e7b.download",
  "time before context deadline": "23h59m59.999994619s"
}
2022-08-11 02:07:57.101 [http-transport]: existing file size
{
  "file size": 0,
  "deal size": 374105
}
2022-08-11 02:07:57.109 [http-transport]: libp2p-http url
{
  "url": "libp2p://12D3KooWCVXs8P7iq6ao4XhfAmKWrEeuKFWCJgqe9jGDMTqHYBjw",
  "peer id": "12D3KooWCVXs8P7iq6ao4XhfAmKWrEeuKFWCJgqe9jGDMTqHYBjw",
  "multiaddr": "/ip4/139.178.68.217/tcp/6745"
}
2022-08-11 02:07:57.116 [http-transport]: started async http transfer
2022-08-11 02:07:57.116 [http-transport]: sending http request
{
  "received": 0,
  "remaining": 374105,
  "range-rq": "bytes=0-"
}
2022-08-11 02:07:57.731: transfer progress
{
  "bytes received": 53067,
  "deal size": 374105,
  "percent complete": 14
}
2022-08-11 02:07:57.758: transfer progress
{
  "bytes received": 85433,
  "deal size": 374105,
  "percent complete": 22
}
2022-08-11 02:07:57.909: transfer progress
{
  "bytes received": 118386,
  "deal size": 374105,
  "percent complete": 31
}
2022-08-11 02:07:57.935: transfer progress
{
  "bytes received": 151154,
  "deal size": 374105,
  "percent complete": 40
}
2022-08-11 02:07:57.943: transfer progress
{
  "bytes received": 216690,
  "deal size": 374105,
  "percent complete": 57
}
2022-08-11 02:07:58.087: transfer progress
{
  "bytes received": 249458,
  "deal size": 374105,
  "percent complete": 66
}
2022-08-11 02:07:58.095: transfer progress
{
  "bytes received": 261911,
  "deal size": 374105,
  "percent complete": 70
}
2022-08-11 02:07:58.138: transfer progress
{
  "bytes received": 314994,
  "deal size": 374105,
  "percent complete": 84
}
2022-08-11 02:07:58.143: transfer progress
{
  "bytes received": 347762,
  "deal size": 374105,
  "percent complete": 92
}
2022-08-11 02:07:58.143 [http-transport]: http server sent EOF
{
  "received": 374105,
  "deal-size": 374105
}
2022-08-11 02:07:58.151: transfer progress
{
  "bytes received": 374105,
  "deal size": 374105,
  "percent complete": 100
}
2022-08-11 02:07:58.158 [http-transport]: http transfer completed successfully
2022-08-11 02:07:58.175 [http-transport]: http request finished successfully
{
  "nBytesReceived": 374105,
  "file size": 374105
}
2022-08-11 02:07:58.182: deal data-transfer completed successfully
{
  "bytes received": 374105,
  "time taken": "1.089111576s"
}
2022-08-11 02:07:58.187: checking commP
2022-08-11 02:07:59.004: commP matched successfully: deal-data verified
2022-08-11 02:07:59.042: updated deal checkpoint in DB
{
  "old checkpoint": "Accepted",
  "new checkpoint": "Transferred"
}
2022-08-11 02:07:59.049: deal data transfer finished successfully
2022-08-11 02:07:59.055: deal data-transfer can no longer be cancelled
2022-08-11 02:07:59.072: sending deal to deal publisher
2022-08-11 02:27:59.507: deal paused because of recoverable error
{
  "err": "failed to publish deal d1e56dbf-fe49-4307-bd6a-65c488d93e7b: publish validation failed: simulating deal publish message: apply message failed: implicit message failed with exit code: 16 and error: message failed with backtrace:\n00: f05 (method 4) -- All deal proposals invalid (16)\n01: f06 (method 5) -- Verified Dealsize 524288 is below minimum in usedbytes (16)\n (RetCode=16)",
  "checkpoint": "Transferred",
  "retry": "manual"
}
2022-08-11 02:27:59.749: deal go-routine finished execution
2022-08-11 10:47:11.526: user manually terminated the deal
{
  "err": "failed to publish deal d1e56dbf-fe49-4307-bd6a-65c488d93e7b: publish validation failed: simulating deal publish message: apply message failed: implicit message failed with exit code: 16 and error: message failed with backtrace:\n00: f05 (method 4) -- All deal proposals invalid (16)\n01: f06 (method 5) -- Verified Dealsize 524288 is below minimum in usedbytes (16)\n (RetCode=16)"
}
2022-08-11 10:47:11.648: cleaning up deal
2022-08-11 10:47:11.725: deal finished
2022-08-11 10:47:12.144: untagged funds for deal as deal finished
{
  "untagged publish": "1050000000000000000",
  "untagged collateral": "110400374015",
  "err": null
}
2022-08-11 10:47:12.548: untagged storage space for deal
2022-08-11 10:47:12.671: finished cleaning up deal
```

**Comment by en0ma** [comment]
closing this as this has been resolved
